build_from.py

```diff
@@ -92,8 +92,12 @@ def generate_diffs(reg_tree, act_title_and_section, builder, layer_cache):
     diffs.add_argument('--no-generate-diffs', dest='generate_diffs',
                        action='store_false', help="Don't generate diffs")
     diffs.set_defaults(generate_diffs=True)
-    parser.add_argument('--checkpoint', required=False,
+    parser.add_argument('--checkpoint', dest='checkpoint_dir', required=False,
                         help='Directory to save checkpoint data')
+    parser.add_argument(
+        '--version-identifier', dest='doc_number', required=False,
+        help=('Do not try to derive the version information. (Only use if '
+              "your regulation is older than federalregister.gov's records)"))
```

Comment:
Roger, went with:
```
  --version-identifier DOC_NUMBER
                        Do not try to derive the version information. (Only
                        use if the regulation has no electronic final rules on
                        federalregister.gov, i.e. has not changed since before
                        ~2000)
```